install/pre-install-payload/containerd/payload.sh

```diff
@@ -22,10 +22,12 @@ function setup_env_for_arch() {
 		"linux/amd64") 
 			image="docker.io/library/centos"
 			kernel_arch="x86_64"
+			golang_arch="amd64"
 			;;
 		"linux/s390x")
 			image="docker.io/library/clefos"
 			kernel_arch="s390x"
+			golang_arch="amd64"
 			;;
 		(*) echo "$1 is not supported" && exit 1
```

Comment:
a nit: error message should be redirected to `stderr`

Reply:
Adjusted, thanks @hbrueckner!